manager/integration/tests/test_node.py

```diff
@@ -350,6 +350,27 @@ def test_replica_scheduler_no_disks(client):  # NOQA
     common.wait_for_volume_delete(client, vol_name)
 
 
+@pytest.mark.node  # NOQA
+def test_replica_zone_anti_affinity(client):  # NOQA
+    """
+    Test replica scheduler with zone anti-affinity
+
+    1. Set zone anti-affinity to hard, and replica count to 2.
+    2. Create 2 nodes and add the same zone label to them.
+    3. Create a volume.
+    4. Wait for volume condition `scheduled` to be false.
+    5. Change the zone label to be different on one node.
+    6. Create a volume.
+    7. Wait for volume condition `scheduled` to be success.
+    8. Clear the volume.
+    9. Set zone anti-affinity to soft.
```

Comment:
After we set `zone anti-affinity` to soft, we can increase the number of volume replicas to e.g 5 and check replicas scheduled successfully and in healthy state.

Reply:
I think 3 is good enough. We can set the replicas to 5, but in this case we need to make sure we have the node soft anti-affinity is set as well.